asapdiscovery-dataviz/asapdiscovery/dataviz/html_viz.py

```diff
@@ -152,11 +153,15 @@ def get_html_footer(self):
         """
         Get HTML footer for pose visualization
         """
-        if self.target == "sars2":
-            return colour_sars2 + orient_tail_sars2
-        elif self.target == "mers":
-            return colour_mers + orient_tail_mers
-        elif self.target == "7ene":
-            return colour_7ene + orient_tail_7ene
-        elif self.target == "272":
-            return colour_mers + orient_tail_272
+        if self.target == "sars2_mpro":
+            return colour_sars2_mpro + orient_tail_sars2_mpro
+        elif self.target == "mers_mpro":
+            return colour_mers_mpro + orient_tail_mers_mpro
+        elif self.target == "7ene_mpro":
+            return colour_7ene_mpro + orient_tail_7ene_mpro
+        elif self.target == "272_mpro":
+            return colour_mers_mpro + orient_tail_272_mpro
+        else:
+            raise ValueError(
```

Comment:
We are missing an HTML viz angle for MAC1, it might be good to add one in now so we can make poses, otherwise pose-making stage will fail.

Reply:
I'll drop this in!